src/cti_authoring_tool/src/components/Controls/Fields/ComplexTableField.vue

```diff
@@ -38,12 +38,8 @@ import { ComplexTableProperty, FormattedText, Property, Sort, SummaryParser, Tab
 // Components
 import TabularField from "./TabularField.vue";
 
-// TODO: Implement collapse / uncollapse all when table header clicked
-
 // TODO: Implement abridged summary for tables with 100+ rows
```

Comment:
Remove any remaining TODOs -- if you still want to do them, put an issue in the jira backlog.

Reply:
Removed 👍